storybook/preview-head.html

```diff
@@ -1 +1,4 @@
-<link href="https://fonts.googleapis.com/css?family=Montserrat:400,600" rel="stylesheet">
\ No newline at end of file
+<link
+  href="https://fonts.googleapis.com/css?family=Montserrat:500,700"
```

Comment:
Was this just missed with the initial v2 changes? Those font weight diffs are not new.

Reply:
correct, storybook just had the wrong weights. DSv2 changed 400,600 to 500,700